README.md

```diff
@@ -326,6 +326,7 @@ The [Fediverse](https://en.wikipedia.org/wiki/Fediverse) is the place of hope fo
 - [Golden Cheetah](https://www.goldencheetah.org/) [<img src="https://raw.githubusercontent.com/humanetech-community/awesome-humane-tech/main/logo/github.svg?sanitize=true" width="16"/>](https://github.com/goldencheetah/goldencheetah) - Performance software for Cyclists, Runners and Triathletes that reads from many devices.
 - [gammastep](https://gitlab.com/chinstrap/gammastep) - Linux tool to adjust the color temperature of your screen to your surroundings.
 - [MediLog](https://gitlab.com/toz12/medilog) - Android app to capture medical data, e.g. Blood Pressure and Weight as well as a simple Diary.
+- [Energize <img src="https://raw.githubusercontent.com/humanetech-community/awesome-humane-tech/main/logo/codeberg.svg?sanitize=true" width="16"/>](https://codeberg.org/epinez/Energize) - Android app for tracking your food and nutrition easily and fast.
```

Comment:
The convention on this list is that only in the combination of a site and a repo, is the repo software icon displayed. I understand the passion for Codeberg, and have an account myself, but use of the icon is inappropriate here. Please format similar to the entry just above.

Reply:
Ah sorry, did not get that. I thought it was just indicating where the repo is located. Fixed!